src/hardware/developing-an-iox/protocol.md

```diff
@@ -237,6 +238,7 @@ Sent from the IOX to the GO device when the IOX wants create a log that cannot f
 ### IOX Request/Status (0x25)
 
 Sent from the IOX to the GO device to inform the GO device of events or status changes.
+which is then followed by [RX Data (0x0C)](#rx-data-0x0c).
```

Comment:
How about `A message of this type is to be added before the start of a [0x0C](#rx-data-0x0c) message series, and also after the end of it.` ?